front/src/styles/scss/applications/operationalStudies/_scenario.scss

```diff
@@ -118,6 +118,56 @@
         img {
           height: 1rem;
         }
+        .scenario-details-infra-loading-state {
+          font-size: 0.9rem;
+          display: flex;
+          align-items: center;
+          margin-right: 0.5rem;
+
+          .infra-loader {
+            width: 0.7rem;
+          }
+          .infra-loader:first-child {
+            animation: dot-flashing 0.5s infinite linear alternate;
+            animation-delay: 0s;
+          }
+          .infra-loader:nth-child(2) {
+            animation: dot-flashing 0.5s infinite linear alternate;
+            animation-delay: 0.25s;
+            margin-inline: -0.2rem;
+          }
+          .infra-loader:nth-child(3) {
+            animation: dot-flashing 0.5s infinite linear alternate;
+            animation-delay: 0.5s;
+          }
+
+          @keyframes dot-flashing {
+            0% {
+              color: var(--orange);
+            }
+            50%,
+            100% {
+              color: #f8e3d8;
+            }
+          }
+
+          .steps {
+            font-weight: bold;
+            margin-left: 0.5rem;
+            margin-top: 0.1rem;
+          }
+        }
+        .cached {
+          color: var(--green);
+        }
+        .loading {
+          color: var(--orange);
+        }
+      }
+      .scenario-details-infra-error {
+        color: var(--red);
+        font-weight: bold;
```

Comment:
Prefer `500` instead of `bold`, more elegant, thx !

Reply:
done